### DISCUSSION TITLE
Google Drive out of space because of old models in trash

### DISCUSSION THREAD

**Discussion by estaesta** [voicepaw/so-vits-svc-fork#375] [Q&A] [answered]
When training on Colab, the old models and lightning ckpt are moved to the drive trash instead of being permanently deleted. Files that are placed in the trash are still counted towards the storage limit. Is there any way to automatically delete the old models permanently so that they don't take up storage space?
I have lost my training results several times because my drive was full due to old models that were still in the trash.

**Comment by Meldoner** [comment]
It's really annoying

**Comment by estaesta** [comment]
One of the workarounds is to overwrite the file with an empty file before deleting it, but this method is noticeably slower.

**Comment by 34j** [comment, marked as answer]
Migrate to #474
